### ISSUE TITLE
Add Final Quest Cultural Goods Requirements for Settlements Resource Overview

### ISSUE THREAD

**Issue by SteveTheTechie** [mainIine/foe-helfer-extension#2307]
In the cultural settlements, there is always a final story quest that requires cultural goods above and beyond what is required for all the embassy advancements.  However, this is not included in the FOE Helper Resource Overview.  It would be great if this additional cultural goods requirement could be added to the Resource Overview (add it at the bottom) to give a truer picture of what amounts of cultural goods are really needed to complete the cultural settlements.  Currently, I have to remember to add that on top of what is shown in the Resource Overview.  I believe the amounts for the final quests are known and fixed, so I believe this addition should be possible.

**Comment by Beelzebob6666** [comment]
I also requested that some time ago... 

main problem with that is that those would need to be hardcoded as the info is not available from the game data until the quest is finally coming up.

But I might take a look into it.

**Comment by Paladiac** [comment]
I am confused by this request.  The final quest does not require cultural goods to be paid to advance or finish the quest - it requires the goods to be produced - very different than what the goods needed for unlocking the various stages.  Seems to me that adding this would only confuse things - because then users would think they can produce those goods BEFORE they even reach that final quest, which is wrong.  They need to produce (or more acurately 'collect') these goods AFTER the quest is obtained.

**Comment by Gindi4711** [comment]
You are correct: The player has to save the resources shown by foe helper first, then unlock technologies and do the quests and after that produce the goods for the last quest.

The only situation where it could make sense is secondary resources needed to produce these goods.
For Egypt it would be smart to save the loot for the last goods in advance.
For all other cultures it is more complicated. In an ideal situation you would save up coins for the first 8h production and in the meantime produce more coins for the following goods productions, but there may be different strategies here.

But required coins/loot is of limited use anyway because it does not take the chance of x4 production into account and always shows the worst case with no x4 production, but showing average values is not a good idea either

**Comment by SteveTheTechie** [comment]
Ok, but the way it is now kind of gives an incomplete picture, as the player must produce additional goods beyond what is currently represented in FOE Helper to complete the final quest.  I do understand that this is different than what is needed for the embassy advancements, so maybe just adding it as a separate section/line item at the bottom of the resource screen would suffice.  Right now users have to go look this information up in one of the wikis which seems a bit silly since you already have a lot of the resource information in FOE Helper.

In other words, maybe it does not exactly fit with what you had envisioned for the resource overview screen, but it would still be useful nonetheless, so maybe just put it in a separate section below the current resource information.
